api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2ConnectorSpec.java

```diff
@@ -14,7 +14,7 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"tasksMax", "config"})
+@JsonPropertyOrder({"tasksMax", "autoRestartConnectorsAndTasks", "config"})
```

Comment:
Should this still be needed? I guess it now lives inside the connector specifications.